librecad/src/main/qc_applicationwindow.cpp

```diff
@@ -3664,3 +3670,8 @@ void QC_ApplicationWindow::updateWindowTitle(QWidget *w)
         w->setWindowTitle(w->windowTitle()+m_qDraftModeTitle);
     }
 }
+
+void QC_ApplicationWindow::goto_wiki()
+{
+    QDesktopServices::openUrl(QUrl("http://wiki.librecad.org/index.php/Main_Page"));
```

Comment:
Keep the URL as short as possible:

`QUrl("http://wiki.librecad.org/")`

When we move the Wiki, the direct page link becomes invalid.